baselines/dom.generated.d.ts

```diff
@@ -17230,7 +17230,8 @@ interface Window extends EventTarget, AnimationFrameProvider, GlobalEventHandler
     readonly pageXOffset: number;
     /** @deprecated This is a legacy alias of `scrollY`. */
     readonly pageYOffset: number;
-    readonly parent: WindowProxy | null;
+    /** Refers to either the parent WindowProxy, or itself. If you create a child context, then remove the parent you could get null though. */
```

Comment:
What about:

> Refers to either the parent WindowProxy, or itself.
>
> It can rarely be null e.g. for `window` for an iframe that is already removed from the parent.

Reply:
Having it multiline doesn't really work because it's in two places with different indentations, so I made this a single line

Reply:
I think `\n` automatically does the indentation.

Reply:
e.g. postMessage